modules/storage-persistent-storage-pv.adoc

```diff
@@ -145,6 +145,7 @@ the Pods that use these volumes are deleted.
 |iSCSI  | ✅ | ✅ |  -
 |Local volume | ✅ | - |  -
 |NFS  | ✅ | ✅ | ✅
+|Red Hat OpenShift Container Storage | ✅ | ✅ | ✅
```

Comment:
Need OCS QE to confirm that it support access mode RWO,ROX,RWX

Reply:
@ebenahar or @RazTamir Could one of you PTAL and confirm with `LGTM`? Thanks.

Reply:
ROX is not supported

Reply:
@RazTamir Removed ROX. Look good?

Reply:
Yes, thank you.
LGTM